_posts/WayneHoman/2016-01-12-Homan-Wayne-post.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: post
 author: WayneHoman
+title: Nope, try again
```

Comment:
@wfh1972 I believe this comma will make Jekyll mad.  Try enclosing it in `"`. 

Also I think your first post didn't have an error- it just wasn't showing up because of the site build problems we were having.